### PR TITLE
Final tutorial level

### LINKED ISSUE #102: Tutorial 2.0
Make the tutorial level a tutorial level.

* Easy. Don't let them die
* Introduce every control with an easy obstacle
* Images for controls

**Estimated:** 45 **Min:** 30 **Max:** 60

### PR DESCRIPTION
**Pull request by clayton-snyder**
Fixes #102 (WIP)

* Jump, strafe, and killzone introduced first; have to jump to avoid the lava and get to the next platform

* Shooting grapple and rappelling up/down introduced next; have to shoot the point and rappel up to get to the next platform.

* Swinging/releasing introduced next; have to swing over the next wall to move on

* Bouncy platforms introduced next. Can bounce over the next wall or use the grapple point to get across.

* Lower half of level allows the player to bounce around and play with chain-swinging on grapple hooks. Bouncy floor makes it easier and faster to move around so players can more conveniently practice the swing mechanic before they move on.

**Estimated:** 45 **Min:** 30 **Max:** 60 **Actual:** Not finished yet

### CODE REVIEW

**Comment by clayton-snyder** [comment]
Need to make sure that there is no wall jump possibility.

The first wall that's too high to jump over has this issue.

**Comment by clayton-snyder** [comment]
add a button on the main menu to access the level

**Comment by Chris-Johnston** [comment]
Awesome. I think that this level is the right level of difficulty for the tutorial (and that's none at all).

I would remove the "you win" sprite that appears when the player wins, since we are immediately loading the next scene as soon as they win.

I was able to notice it for a few frames before the next scene loaded.

File: 385/ProjectSettings/EditorBuildSettings.asset [hunk 1]
```diff
@@ -41,4 +41,7 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/Scenes/TutorialFinal.unity
     guid: 111a8aacc7a8148c7911ee4fb6676c42
+  - enabled: 1
+    path: Assets/Scenes/RealFinalTutorial.unity
```
Comment (Chris-Johnston) [inline 385/ProjectSettings/EditorBuildSettings.asset]: That naming scheme tho.

**Comment by Chris-Johnston** [comment]
I would move the climing instruction towards the end when climbing is needed, or at least space it out from the point it's next to. I would also add some label that explains what the red and the blue are. Otherwise I think that the images are a good improvement.

**Comment by clayton-snyder** [comment]
Idea was to have a wall too high at the beginning so they had to climb up to get there, but there does seem to be too many instructions crowded at the beginning.

So I moved the climb instruction to the end and changed the rest of the level so they shouldn't need to know how to climb to get past the parts before the end.

Also added labels to win/lose zones.

**Comment by SaamIAm** [comment]
Adjust the camera so when 'c' is pressed I can see the entire level and not have portions of it cut off.

**Comment by Chris-Johnston** [comment]
LGTM :shipit:

**Pull request merged**